pom.xml

```diff
@@ -93,7 +93,7 @@
         <javase.version>1.8</javase.version>
         <jakartaee.api.version>8.0.0</jakartaee.api.version>
         <javax.security.enterprise.version>1.1-b01.payara-p3</javax.security.enterprise.version>
-        <nimbus-jose-jwt.version>8.2.1</nimbus-jose-jwt.version>
+        <nimbus-jose-jwt.version>8.19</nimbus-jose-jwt.version>
```

Comment:
Is this meant to be in-line with the version used in the Payara/payara repo?
The one there appears to be version 5.4

Reply:
I upgraded the dependency as in the next step Payara Platform internal Security Connector source will be removed and replaced by ecosystem-security-connectors.